core/services/registrysyncer/orm.go

```diff
@@ -0,0 +1,277 @@
+package registrysyncer
+
+import (
+	"context"
+	"crypto/sha256"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"math/big"
+	"strconv"
+
+	"github.com/smartcontractkit/libocr/ragep2p/types"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities"
+	"github.com/smartcontractkit/chainlink-common/pkg/sqlutil"
+
+	kcr "github.com/smartcontractkit/chainlink/v2/core/gethwrappers/keystone/generated/capabilities_registry"
+	"github.com/smartcontractkit/chainlink/v2/core/logger"
+	p2ptypes "github.com/smartcontractkit/chainlink/v2/core/services/p2p/types"
+)
+
+type capabilitiesRegistryDON struct {
+	ID               uint32   `json:"id"`
+	ConfigVersion    uint32   `json:"configVersion"`
+	F                uint8    `json:"f"`
+	IsPublic         bool     `json:"isPublic"`
+	AcceptsWorkflows bool     `json:"acceptsWorkflows"`
+	Members          []string `json:"members"`
+}
+
+type capabilitiesRegistryDONInfo struct {
+	capabilitiesRegistryDON
+	CapabilityConfigurations map[string]capabilities.CapabilityConfiguration `json:"capabilityConfigurations"`
+}
+
+type capabilitiesRegistryNodeInfo struct {
+	NodeOperatorId      uint32   `json:"nodeOperatorId"`
+	ConfigCount         uint32   `json:"configCount"`
+	WorkflowDONId       uint32   `json:"workflowDONId"`
+	Signer              string   `json:"signer"`
+	P2pId               string   `json:"p2pId"`
+	HashedCapabilityIds []string `json:"hashedCapabilityIds"`
+	CapabilitiesDONIds  []string `json:"capabilitiesDONIds"`
+}
+
+type capabilitiesRegistryCapabilityInfo struct {
+	ID             string `json:"id"`
+	CapabilityType int    `json:"capabilityType"`
+}
+
+func (t *LocalRegistry) MarshalJSON() ([]byte, error) {
+	idsToDONs := make(map[string]capabilitiesRegistryDONInfo)
```

Comment:
Same as above, just use the types we've introduced for DON, Capability in the registrysyncer package (I know I told you to copy these types here, but that was when we were using the generated gethwrapper types)

Reply:
Please see my previous comment on the matter. @cedric-cordenier